notebooks/205-vision-background-removal/205-vision-background-removal.ipynb

```diff
@@ -57,12 +71,36 @@
     "import numpy as np\n",
     "import torch\n",
     "from IPython.display import HTML, FileLink, display\n",
-    "from model.u2net import U2NET, U2NETP\n",
     "from openvino.runtime import Core\n",
     "from openvino.tools import mo"
    ]
   },
   {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "57edabfb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import local modules\n",
+    "\n",
+    "utils_file_path = Path(\"../utils/notebook_utils.py\")\n",
+    "notebook_directory_path = Path(\".\")\n",
+    "\n",
+    "if not utils_file_path.exists():\n",
+    "    !git clone --depth 1 https://github.com/openvinotoolkit/openvino_notebooks.git\n",
+    "    utils_file_path = Path(\"./openvino_notebooks/notebooks/utils/notebook_utils.py\")\n",
+    "    notebook_directory_path = Path(\"./openvino_notebooks/notebooks/205-vision-background-removal/\")\n",
+    "\n",
+    "sys.path.append(str(utils_file_path.parent))\n",
+    "sys.path.append(str(notebook_directory_path))\n",
+    "\n",
+    "from notebook_utils import load_image\n",
+    "from model.u2net import U2NET, U2NETP"
```

Comment:
Why clone the whole repository? The `notebook_utils.py` and `u2net.py` can be downloaded with urllib, like in other notebooks.

Reply:
205 and 405 notebooks rely on one file each. Only 407 is the problem. We could leave this as it is for now, but fix it next time.

Reply:
In my opinion, cloning the repo is a cleaner and more reliable approach than retrieving a file with a URL request. I only did this for the utils module to save users' data bandwidth